xla/debug_options_flags.cc

```diff
@@ -1095,6 +1096,15 @@ void MakeDebugOptionsFlags(std::vector<tsl::Flag>* flag_list,
       debug_options->xla_gpu_collective_inflation_factor(),
       "Inflation factor for collectives. If set to > 1, each XLA/GPU "
       "collective will execute multiple times (will yield incorrect results)"));
+
+  flag_list->push_back(tsl::Flag(
+      "xla_llvm_split_module_level",
+      int32_setter_for(&DebugOptions::set_xla_llvm_split_module_level),
+      debug_options->xla_llvm_split_module_level(),
+      "Decide how llvm modules are split: Whether to force inline or preserve "
+      "locals. 0 = no inline, 1 = force inline before split, 2 = no line and "
+      "PreserveLocals=False during split, default is 0"));
```

Comment:
Why isn't 1 the better default?

Reply:
Was hoping google can benchmark their internal workloads to make sure it does not break anything so we can make it default.

Reply:
I will rerun experiments to ensure the slowdown is actually there not because i was competing the dgx resources with someone else. Compiler team do mention that by not doing inline, some optimization opportunities are not available in ptxas (for example, loop invariant analysis would not move dowork() outside any loops, alias analysis will become more conservative). 

Another thing is that I was told maybe in some workloads people are willing to sacrifice some perf for faster compilation. But for sure, the 1.4x slowdown is too fast. I should be able to run this on other models as well to get more info on this option.